datasets/batch_view/crash_summary/reference.md

```diff
@@ -38,7 +38,7 @@ The DAG is [here](https://github.com/mozilla/telemetry-airflow/blob/master/dags/
 ## Schema
 
 ```
-root
+root                                                                            
```

Comment:
Nit: can you remove the trailing whitespace here?

Reply:
Ah, sorry, results of editing via web GUI...